test/api/v1/rpc_test.go

```diff
@@ -0,0 +1,176 @@
+package api_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/talis/internal/db/models"
+	"github.com/celestiaorg/talis/pkg/api/v1/handlers"
+	"github.com/celestiaorg/talis/test"
+)
+
+var defaultProjectCreateParams = handlers.ProjectCreateParams{
+	Name:        "test-project",
+	Description: "A test project",
+	Config:      `{"resources": {"cpu": 2, "memory": "4GB"}, "settings": {"env": "test"}}`,
+}
+
+var defaultTaskGetParams = handlers.TaskGetParams{
+	ProjectName: "test-project",
+	TaskName:    "test-task",
+}
+
+func TestProjectRPCMethods(t *testing.T) {
+	suite := test.NewSuite(t)
+	defer suite.Cleanup()
+
+	// Create a project directly in the database
+	project := models.Project{
+		OwnerID:     models.AdminID,
+		Name:        defaultProjectCreateParams.Name,
+		Description: defaultProjectCreateParams.Description,
+		Config:      defaultProjectCreateParams.Config,
+	}
+	err := suite.ProjectRepo.Create(suite.Context(), &project)
```

Comment:
This should be `APIClient.CreateProject` right?

Reply:
oh yes, thanks for pointing it out. will fix it quickly

Reply:
done